src/components/NavBar.tsx

```diff
@@ -0,0 +1,19 @@
+
+import React from 'react';
+import { BrowserRouter as Router, Switch, Route, Link } from 'react-router-dom';
+
+function NavBar() {
```

Comment:
we could just use 
```suggestion
export function NavBar() {
```
and not export at the bottom :)